### PR TITLE
Fix: Change memcached's maintenance_policy duration attribute type.

### PR DESCRIPTION
**Pull request by r4rohan**
The duration attribute of the maintenance policy for Memcached is expected a string not a number. [Doc Link](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/memcache_instance#duration)

Upon passing the number in Memcached maintence_policy, terraform is throwing an Illegal duration format error because it's expecting the value to end with 's', for example, '10800s'. Refer to the attached screenshot.
Screenshot 2023-03-16 at 4 35 06 PM

Since the value type is set as a number right now which means it will not expect string and throws  The given value is not suitable for module.memcached_memorystore.var.maintenance_policy.

Added maintenance_policy block in the example.

Note: The code is tested and working. It's able to create a Memcached memstore with a maintenance policy with duration.

### CODE REVIEW

**Comment by aembley** [comment]
This is a blocker for my use of the module. Any chance it can be merged soon?

**Comment by apeabody** [comment]
Thanks @r4rohan - Looks like the LINT just needs a `terraform fmt`.

**Comment by r4rohan** [comment]
@apeabody formatted

File: examples/memcache/main.tf [hunk 1]
```diff
@@ -26,13 +26,23 @@ module "private-service-access" {
 
 module "memcache" {
   source             = "../../modules/memcache"
-  name               = "example-memcache"
+  name               = "exammple-memcache"
```
Comment (apeabody) [inline examples/memcache/main.tf:29]: Typo?

```suggestion
  name               = "example-memcache"
```

File: examples/memcache/variables.tf [hunk 2]
```diff
@@ -17,4 +17,5 @@
 variable "project_id" {
   description = "The ID of the project in which the resource belongs to."
   type        = string
+  default     = "rohan-orbit"
```
Comment (apeabody) [inline examples/memcache/variables.tf:20]: nit: might want to use a more generic value

**Comment by r4rohan** [comment]
@apeabody I'm closing this PR and creating a new one due to the wrong email id push.

**Pull request closed without merging**